bin/rake

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
+load File.expand_path("spring", __dir__)
```

Comment:
sneaked in from what i though was only a local change attempt to speed up rspec and rubocop - can remove